internal/provider/resource_cloudflare_teams_accounts_test.go

```diff
@@ -66,6 +68,8 @@ resource "cloudflare_teams_account" "%[1]s" {
     header_text = "hello"
     logo_path = "https://example.com"
     background_color = "#000000"
+	mailto_subject = "hello"
```

Comment:
these fields will need to be added to the schema to make them usable - https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/internal/provider/schema_cloudflare_teams_accounts.go

Reply:
bah, good call. Rushed this one.